### ISSUE TITLE
Support for `f90-mode`

### ISSUE THREAD

**Issue by zbeekman** [editorconfig/editorconfig-emacs#200]
Hi, first of Great project!!!!

To get support for Fortran, you need something like:

```elisp
  (add-to-list 'editorconfig-indentation-alist
               '(f90-mode
                 f90-associate-indent f90-continuation-indent
                 f90-critical-indent f90-do-indent f90-if-indent
                 f90-program-indent f90-type-indent))
```

It would be great if this was baked in out of the box. Thanks!

P.S. I created a project to use GitHub Actions to validate PRs and pushes against a project's `.editorconfig`. Check it out on GitHub Marketplace (don't worry, it's free as in beer and freedom): https://github.com/marketplace/actions/editorconfig-action

**Comment by xuhdev** [comment]
Thanks, it looks awesome!

Could you make a PR? You only need to add a line below https://github.com/editorconfig/editorconfig-emacs/blob/d073dd44c4813536dee6eadaf9e43f36ac0a3202/editorconfig.el#L159

**Comment by xuhdev** [comment]
Closed by #201
